tbx/static_src/sass/components/_image.scss

```diff
@@ -36,3 +36,16 @@
         font-weight: $weight--semibold;
     }
 }
+
+.wide-image {
+    &__caption,
+    &__attribution {
+        @include media-query(large) {
+            margin-right: $spacer-medium;
```

Comment:
A quick comment to explain the margin-right here might be helpful - otherwise looks good.